src/features/list-view/ListView.js

```diff
@@ -28,14 +30,22 @@ type CellRendererArgs = {|
 
 class ListView extends React.PureComponent<Props> {
     cellRenderer = ({ columnIndex, key, rowIndex, style }: CellRendererArgs) => {
-        const { getGridCell, getGridHeader } = this.props;
+        const { getGridCell, getGridHeader, getGridHeaderSort, onSortChange } = this.props;
 
         if (rowIndex === 0) {
             const displayName = getGridHeader(columnIndex);
+            const sortIcon = getGridHeaderSort && getGridHeaderSort(columnIndex);
             return (
-                <div className="list-view-column-header" key={key} style={style}>
+                <button
+                    className="list-view-column-header"
+                    key={key}
+                    style={style}
+                    type="button"
+                    onClick={() => onSortChange && onSortChange(columnIndex)}
```

Comment:
Shouldn't onSortChange pass the direction? The method signature feels like it's missing something.

Reply:
I don't think onSortChange needs to pass the direction since the direction is found off of the column in props.columns in MetadataViewPage.

In MetadataViewPage, I have a check that finds ```direction``` off of the column with columnIndex, then that gets passed into the reducer.